tests/ui/package.json

```diff
@@ -9,11 +8,20 @@
     "ts-node": "^10.9.1"
   },
   "devDependencies": {
-    "@playwright/test": "^1.41.2"
+    "@cucumber/cucumber": "^10.3.1",
+    "@playwright/test": "^1.41.2",
+    "allure-commandline": "^2.27.0",
+    "allure-cucumberjs": "^2.12.0",
+    "allure-js-commons": "^2.12.0",
+    "allure-playwright": "^2.12.0"
   },
   "scripts": {
+    "build": "tsc -p tsconfig-build.json",
     "pretest": "npx ts-node src/utilities/init.ts",
-    "test": "cross-env test_env=dev cucumber-js test",
-    "test_cmd_line": "cucumber-js test"
+    "test": "npm run build && cross-env test_env=dev cucumber-js test",
+    "tests:tagged": "node ./node_modules/.bin/cucumber-js  --tags",
+    "test_cmd_line": "cucumber-js test",
+    "allure": "npx allure generate allure-results -c -o allure-reports",
```

Comment:
i thought that we were going for the single file version of the report. That would remove the need to serve and allow for the report to be share more easily

Reply:
Fixed